src/main/scala/edu/colorado/plv/cuanto/jsy/common/OpParserLike.scala

```diff
@@ -103,16 +102,29 @@ trait OpParserLike extends RegexParsers with RichParsers {
     binaryOps(bop)
   }
 
-  /** Parser for unary expressions. */
+  /** Yields a parser for unary expressions with an instantiation for `uop`
+    *
+    * @see uop
+    */
   def unary: Parser[Expr] =
     positioned {
       uop ~ unary ^^ { case op ~ e => Unary(op, e) }
     } |
     atom
 
-  /** Parser for atoms. */
+  /** Parse parenthesized expressions and delegates to `opatom`.
+    *
+    * @see opatom
+    */
   def atom: Parser[Expr] =
     opatom |
-    "(" ~> expr <~ ")" |
+    parenthesized |
+    block |
     failure("expected an atom")
+
+  def parenthesized: Parser[Expr] =
+    "(" ~> expr <~ ")"
+
+  def block: Parser[Expr] =
```

Comment:
Why does this abstraction, and the previous, exist here? They seem more generic than simple `ops`, but if they're not used elsewhere I don't think the mental overhead of is worth it.